src/main/java/org/terasology/gooeyDefence/movement/PathfindingManager.java

```diff
@@ -189,8 +189,8 @@ private void calculatePath(JPSConfig config, Consumer<List<Vector3i>> callback)
      */
     private JPSConfig buildJpsConfig(Vector3i start) {
         JPSConfig result = new JPSConfig();
-        result.start = JomlUtil.from(start);
-        result.stop = JomlUtil.from(DefenceField.FIELD_CENTRE);
+        result.start = start;
```

Comment:
I don't know what this result is used for, but it might make sense to create a copy of `start` to store it (`start` can be modified outside, which might yield weird behavior).
```suggestion
        result.start = new Vector3i(start);
```